app/controllers/application_controller.rb

```diff
@@ -1,9 +1,19 @@
 class ApplicationController < ActionController::Base
   before_action :set_am_host
+  before_action :set_page_title
 
   protect_from_forgery with: :exception
   before_filter :configure_permitted_parameters, if: :devise_controller?
 
+  rescue_from StandardError do |exception|
```

Comment:
Thinking of it, wouldn't be better just rescuing `ActiveRecord::RecordNotFound` and `ActionController::RoutingError` whiling letting other exceptions just bubble up? In production mode, Rails will not display exceptions, and I understand the logstash gem will catch errors and log them with the appropriate severity level.